### DISCUSSION TITLE
NewVersion showing url

### DISCUSSION THREAD

**Discussion by PieterL75** [netbox-community/netbox#7548] [General]
Maybe too silly to report a 'bug' for it...
but when there is a new version, the 'version' number contains the url
![image](https://user-images.githubusercontent.com/74899468/137447560-09d445fb-ef1d-4065-8cfc-9bbfa65678a7.png)

Small fix makes it nicer :
![image](https://user-images.githubusercontent.com/74899468/137447665-15be7a52-ed5c-441d-948e-c849ee3496ea.png)

https://github.com/netbox-community/netbox/blob/1f4263aa6d7c4f054f1ccd6fd2249f8314dfa3c7/netbox/netbox/views/__init__.py#L140
replace that with
`'version': str(release_verion),`

Pieter

**Comment by jeremystretch** [comment]
Definitely seems worthy of a bug report. @PieterL75 would you mind opening one? Reusing the screenshots from here would be great.
